src/main/java/games/strategy/triplea/ui/menubar/TripleAMenuBar.java

```diff
@@ -93,36 +94,32 @@ private void createLobbyMenu(final JMenuBar menuBar, final InGameLobbyWatcherWra
     lobby.add(new RemoveGameFromLobbyAction(watcher));
   }
 
-
-  public static File getSaveGameLocationDialog(final Frame frame) {
-
+  /**
+   * Displays a file chooser dialog for the user to select the file to which the current game should be saved.
+   *
+   * @param frame The owner of the file chooser dialog; may be {@code null}.
+   *
+   * @return The file to which the current game should be saved or {@code null} if the user cancelled the operation.
+   */
+  public static File getSaveGameLocation(final Frame frame) {
     // For some strange reason,
     // the only way to get a Mac OS X native-style file dialog
     // is to use an AWT FileDialog instead of a Swing JDialog
     if (SystemProperties.isMac()) {
       final FileDialog fileDialog = new FileDialog(frame);
       fileDialog.setMode(FileDialog.SAVE);
-
-
       fileDialog.setDirectory(ClientContext.folderSettings().getSaveGamePath());
-      fileDialog.setFilenameFilter((dir, name) -> {
-        // the extension should be .tsvg, but find svg extensions as well
-        return name.endsWith(".tsvg") || name.endsWith(".svg");
```

Comment:
This was the only one of three file name filter occurrences that did not include `tsvg.gz` in the list of permissible extensions.  I'm assuming it was simply overlooked in e9f3c06.